tests/datasets/entities/storage_selectors/test_errors.py

```diff
@@ -46,6 +46,9 @@
             None, "geo_region", None, "contexts", "geo.region", nullable=True
         ),
         ColumnToMapping(None, "geo_city", None, "contexts", "geo.city", nullable=True),
+        ColumnToMapping(
```

Comment:
hmm, you shouldn't have to do this, let me see if I can make this change unnecessary